first_responder.md

```diff
@@ -0,0 +1,121 @@
+
+* The idiosyncratic name of the role is intentional.  It is not the duty of the "first responder" to be on-call after hours.  The first responder rotation is an effort to watch and triage production issues in an intentional and organized fashion, since no engineer was officially assigned this responsibility in the past, and such minding of things was haphazard.
+
+## Duties
+
+### Weekly Dependency Updates
+
+#### Ensure dependency update spreadsheet is created
+
+Instructions are here https://docs.google.com/spreadsheets/d/1LysSAPFsRGt9PteWpVswp74xnPXyLLxADpHRfh69VLQ/edit#gid=0
+
+It *may* be that someone in a time zone further east than Palo Alto has already done this, but it is the first responder's responsibility to ensure this gets done.
+
+#### Ensure Monday dependency updates are completed
+
+It is a team task to complete these updates, but the first responder needs to make sure that all codebases needing updates have updates merged and deployed.  Note that some projects need to have PRs created by hand ("Manually clone and update repos that failed bundle update" on line 6 of instructions). It may be helpful to post in channel how many updates each developer should do, given who is working that day and how many PRs there are.
+
+### Verify / Notify coverage for following week
+
+1. Verify first responder for following week is still able to cover it.  Check this on Monday (though it's a good thing to keep in mind throughout the week, in case e.g. unexpected illnesses come up).
+  1. It's the scheduled responder's responsibility to find a swap, not current first responder. _TBD: We'll figure something out if said responder is unavailable due to emergency?_
+  1. The person covering the following week is "on deck" for this week.
+1. Set Slack reminders in `#dlss-infrastructure` for next week's Monday morning. The reminders should indicate who is first responder and who is on deck for that week, and should be set for 3 am Pacific time/6 am Eastern, so that the east coast early risers don't have to wait for it.
+  * documentation on Slack's `/remind` command:  https://get.slack.help/hc/en-us/articles/208423427-Set-a-reminder
+    * e.g. if Alice and Bob are up next week, `/remind #dlss-infrastructure on Monday at 3 am "@alice is the first responder this week, and @bob is on deck"`
+
+### Sign Up for Your Next First Responder Shift
+
+The infrastructure team has 8 developers, so you should be taking a shift every 8 or so weeks.
+* schedule:  https://docs.google.com/spreadsheets/u/1/d/13TJR93Yc9_eF5B7w4XDx6ggG_wb3aLkgCHjpLwmHPBA/
+
+### Proactively Check for Production Problems
+
+See "Proactively Check for Production Problems" section below for specifics. 
+
+### Triage Production Problems
+
+If a user reports a problem, or if one is surfaced from monitoring, the first responder is meant to timebox an investigation of the problem (_TBD: 30 min?_).  See "Triage Production Problems" section for more specifics.
```

Comment:
the rule of thumb for the amount of investigation time is something i'm guessing we'll figure out as we try this, but FYI, open question...